tools/ports/sdl2.py

```diff
@@ -16,6 +16,7 @@ def get(ports, settings, shared):
 
   # get the port
   ports.fetch_project('sdl2', 'https://github.com/emscripten-ports/SDL2/archive/' + TAG + '.zip', SUBDIR)
+  libname = ports.get_lib_name('libSDL2')
```

Comment:
why not lowercase?

Reply:
I'm just following the naming conversions normally used by SDL:
```
$ dpkg -L libsdl2-2.0-0 libsdl2-ttf-2.0-0 | grep .so
/usr/lib/x86_64-linux-gnu/libSDL2-2.0.so.0.8.0
/usr/lib/x86_64-linux-gnu/libSDL2-2.0.so.0
/usr/lib/x86_64-linux-gnu/libSDL2_ttf-2.0.so.0.14.0
/usr/lib/x86_64-linux-gnu/libSDL2_ttf-2.0.so.0
```
Normally when one want to link SDL on UNIX one would use `-lSDL` which corresponds the a library called `libSDL.{a,so}`.   This is also what SDLs build system calls the library.

Reply:
Oh, I see. Yeah, sounds good.